packages/editor/src/components/site-discussion/index.js

```diff
@@ -23,29 +23,16 @@ import { store as editorStore } from '../../store';
 
 const COMMENT_OPTIONS = [
 	{
-		label: (
-			<>
-				{ __( 'Open' ) }
-				<Text variant="muted" size={ 12 }>
-					{ __( 'Visitors can add new comments and replies.' ) }
-				</Text>
-			</>
-		),
+		label: __( 'Open' ),
 		value: 'open',
+		helpText: __( 'Visitors can add new comments and replies.' ),
 	},
 	{
-		label: (
-			<>
-				{ __( 'Closed' ) }
-				<Text variant="muted" size={ 12 }>
-					{ __( 'Visitors cannot add new comments or replies.' ) }
-				</Text>
-				<Text variant="muted" size={ 12 }>
-					{ __( 'Existing comments remain visible.' ) }
-				</Text>
-			</>
-		),
+		label: __( 'Closed' ),
 		value: '',
+		helpText: `${ __(
+			'Visitors cannot add new comments or replies.'
+		) } ${ __( 'Existing comments remain visible.' ) }`,
```

Comment:
These used to be two separate text elements (resulting in 2 separate spans that have `display: block`), which displayed them on multiple lines. This will no longer be the case, and that's potentially undesired.

The difference will manifest in some mobile viewports:

|Before|After|
|---|---|
|![Screenshot 2024-07-23 at 16 57 24](https://github.com/user-attachments/assets/0fd77cd7-18e5-4051-ad80-93372f26c9f7)|![Screenshot 2024-07-23 at 16 59 10](https://github.com/user-attachments/assets/6f93a376-2bdb-495c-8997-611cefd70461)|

The potential issue with that is that translation-wise, it might potentially add confusion. Ideally, we would avoid that change.

On a higher level, the `helpText` prop seems to be a bit limiting since we have evidence that we need more than one string. I wonder if we should take that into consideration while designing it. 

Or perhaps this is not an issue and we're fine with keeping those 2 sentences on the same line. Maybe consult a design person to be sure?

Reply:
Good point. This could be easily achieved if we allowed any `ReactNode` instead of only `string`s (see [related conversation](https://github.com/WordPress/gutenberg/pull/63751#discussion_r1686833039)).

Let's just double-check with @WordPress/gutenberg-design that having the help text for a single option span multiple paragraphs is ok.

Reply:
I don't think that behavior is intentional, it looks a little strange. Help text should be optimised for as few characters as possible. Multiple paragraphs isn't ideal, especially in a radio group that can have it's own separate help text. This constraint feels like an improvement to me :D

Reply:
Sounds good. I'll make sure that the text in question renders without line breaks, regardless of which underlying type we pick for the help text.

Reply:
Thanks @jameskoster! That makes it simple, then - let's go with a single string. Ideally, this should actually be a single string, but changing it to one would invalidate the existing translations, so maybe we can leave with it as you suggested.